### PR TITLE
Add comparison table to EKS page

### PR DESCRIPTION
**Pull request by brikis98**
Fixes [WWW-121](https://gruntwork.atlassian.net/browse/WWW-121): adds a comparison table / "Why Gruntwork" section to the EKS landing page. Note that not all the comparisons paint Gruntwork as the best option, so I'm trying to be honest here. Other comparisons I missed, or corrections to some I got wrong, are very welcome!

* Direct link to EKS page: https://deploy-preview-600--keen-clarke-470db9.netlify.app/eks-for-enterprise/
* Direct link to new section: https://deploy-preview-600--keen-clarke-470db9.netlify.app/eks-for-enterprise/#why-gruntwork

### CODE REVIEW

**Review by yorinasub17 — COMMENTED** [review]
Table design and placement LGTM. I had a bunch of comments where I think we sell Rancher too short.

File: pages/landing/enterprise-eks/index.html [hunk 1]
```diff
@@ -219,6 +223,182 @@
...
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
+      - name: "Service communication ACLs"
+        description: "All communication between services is tightly controlled via service mesh ACLs, out-of-the-box."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
```
[the remark this answers is not in the document]
Comment (brikis98) [reply pages/landing/enterprise-eks/index.html:348]: Hm, I'm seeing Istio available as an option on Rancher, but I'm not sure it's available out-of-the-box... Seems like a bunch of extra work to set up, no?

File: pages/landing/enterprise-eks/index.html [hunk 2]
```diff
@@ -219,6 +223,182 @@
...
+      - name: "CIS EKS Benchmark"
+        description: "The Kubernetes cluster is compliant the CIS EKS Benchmark, out-of-the-box. The CIS EKS Benchmark is an objective, consensus-driven guideline for establishing a secure EKS cluster."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
```
[the remark this answers is not in the document]
Comment (brikis98) [reply pages/landing/enterprise-eks/index.html]: Oh, I missed that. Updated: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/600/commits/79cc2a9ca54d2cc61ca06d2c69146efe0627a8b5

File: pages/landing/enterprise-eks/index.html [hunk 3]
```diff
@@ -219,6 +223,182 @@
...
+      - name: "Pod security policies"
+        description: "Best-practice pod security policies are installed, out-of-the-box."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
```
[the remark this answers is not in the document]
Comment (brikis98) [reply pages/landing/enterprise-eks/index.html]: Updated: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/600/commits/79cc2a9ca54d2cc61ca06d2c69146efe0627a8b5

File: pages/landing/enterprise-eks/index.html [hunk 4]
```diff
@@ -219,6 +223,182 @@
...
+      - name: "CI / CD pipeline for apps"
+        description: "Supports a CI / CD pipeline for rolling out changes to apps out-of-the-box."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
```
[the remark this answers is not in the document]
Comment (brikis98) [reply pages/landing/enterprise-eks/index.html]: Ah, looks like they do now have "Rancher CD". Updated: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/600/commits/79cc2a9ca54d2cc61ca06d2c69146efe0627a8b5

File: pages/landing/enterprise-eks/index.html [hunk 5]
```diff
@@ -219,6 +223,182 @@
...
+        kops: "yes"
+        rancher: "no"
+        eks: "yes"
+        gruntwork: "yes"
+      - name: "Node auto scaling"
+        description: "Supports automatically scaling the number of worker nodes up and down in response to load, out-of-the-box."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
```
[the remark this answers is not in the document]
Comment (brikis98) [reply pages/landing/enterprise-eks/index.html]: Updated: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/600/commits/79cc2a9ca54d2cc61ca06d2c69146efe0627a8b5

File: pages/landing/enterprise-eks/index.html [hunk 6]
```diff
@@ -219,6 +223,182 @@
...
+  - title: Cluster management
+    features:
+      - name: "Manage everything as code"
+        description: "You can manage your Kubernetes cluster as code using tools such as Terraform or CloudFormation."
+        kops: "yes"
+        rancher: "no"
+        eks: "yes"
+        gruntwork: "yes"
+      - name: "Node auto scaling"
```
Comment (ina-stoyanova) [inline pages/landing/enterprise-eks/index.html:267]: Isn't there the "HPA" for EKS that allows auto-scaling?

Comment (brikis98) [reply pages/landing/enterprise-eks/index.html:267]: Yes, but it's not configured out-of-the-box in many of these solutions; it's extra work you have to do.

In other words, it's _possible_ to get most of these features in most of the solutions, but it requires a ton of extra work. 

I tried to make this a tiny bit clearer: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/600/commits/3cdaf5679c38c795e671f2471c457f464054fcfc

File: pages/landing/enterprise-eks/index.html [hunk 7]
```diff
@@ -219,6 +223,182 @@
...
+        description: "The Kubernetes cluster is compliant with the CIS AWS Foundations Benchmark, out-of-the-box. The CIS AWS Foundations Benchmark is an objective, consensus-driven guideline for establishing secure infrastructure on AWS."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
+      - name: "CIS EKS Benchmark"
+        description: "The Kubernetes cluster is compliant the CIS EKS Benchmark, out-of-the-box. The CIS EKS Benchmark is an objective, consensus-driven guideline for establishing a secure EKS cluster."
+        kops: "no"
+        rancher: "yes"
+        eks: "no"
```
Comment (ina-stoyanova) [inline pages/landing/enterprise-eks/index.html:385]: Hm, I'm not a 100% sure if it's out-of-the-box, but I am confused if this is EKS CIS benchmark, isn't EKS also meeting that straightaway? I can't find on AWS's website if they say that explicitly though.

**Review by ina-stoyanova — APPROVED** [review]
Left 2 comments, I think the contents of the table looks good to me!

File: pages/landing/enterprise-eks/_comparison.html [hunk 8]
```diff
@@ -1,6 +1,6 @@
 <div class="col-xs-12">
   <h2>Why Gruntwork?</h2>
-  <p>See how Gruntwork Enterprise EKS compares to other popular options for running Kubernetes in AWS.</p>
+  <p>Here's how the <strong>out-of-the-box experience</strong> with Gruntwork Enterprise EKS compares to the out-of-the-box experience with other popular options for running Kubernetes in AWS:</p>
```
Comment (ina-stoyanova) [inline pages/landing/enterprise-eks/_comparison.html:3]: nice touch!

File: pages/landing/enterprise-eks/index.html [hunk 9]
```diff
@@ -219,6 +223,182 @@
...
+        description: "Includes a web UI for managing apps, nodes, logs, and metrics out-of-the-box."
+        kops: "no"
+        rancher: "yes"
+        eks: "no"
+        gruntwork: "yes"
+      - name: "Pod auto scaling"
+        description: "Supports automatically scaling the number of pods up and down in response to load, out-of-the-box."
+        kops: "no"
+        rancher: "yes"
+        eks: "no"
```
[the remark this answers is not in the document]
Comment (brikis98) [reply pages/landing/enterprise-eks/index.html:315]: See above!

File: pages/landing/enterprise-eks/index.html (hunk 7 above)
Comment (brikis98) [reply pages/landing/enterprise-eks/index.html:385]: No, it doesn't support it out-of-the-box. You have to do a ton of work to meet the benchmark.

**Comment by brikis98** [comment]
Thanks for the reviews! Merging now. Yori, if there are any other corrections (or missing items?), LMK, and I'll do them in follow-up PRs.

File: pages/landing/enterprise-eks/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (yorinasub17) [reply pages/landing/enterprise-eks/index.html:348]: Ah I think you are right. I read through the Istio docs and see that you have to deploy it (from their UI) and then configure. So no out of box experience!

**Pull request merged**